### DISCUSSION TITLE
现在续期已经不行了？运行在github action上面的电报方式

### DISCUSSION THREAD

**Discussion by kilvn** [luolongfei/freenom#220] [Q&A]
主人，出错了。具体是在/home/runner/work/freenoom/freenoom/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php文件的第200行，抛出了一个异常。异常的内容是cURL error 7: Failed to connect to my.freenom.com port 443: Connection timed out (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)，快去看看吧。（账户：xxx@qq.com）

**Comment by luolongfei** [comment]
你这个是超时，你找找看问答区有解决办法。

**Reply by kilvn** [reply]
在railway部署已经没问题了，可能是github action禁止了请求freenom域名。
